linera-views/src/queue_view.rs

```diff
@@ -75,16 +75,16 @@ where
     }
 
     fn flush(&mut self, batch: &mut Batch) -> Result<(), ViewError> {
```

Comment:
Wait, `was_cleared` was not accessed in `flush` previously, so it was not working, right?

Reply:
No.
It is true that the `was_cleared` was not accessed in the `flush`. But the was_cleared is accessed via the `store_count` which uses the `was_cleared` and returns 0 in that case. When there are 0 entries then a process similar to the deletion is occurring.

Reply:
Ok. There was a [discussion]( https://github.com/linera-io/linera-protocol/pull/545#discussion_r1144119422) on the PR that introduced `was_cleared`, but it seems that we never followed up and actually simplified the code. Can we remove `was_cleared` for good now? (The idea is to set `self.front_delete_count = self.stored_indices.len()` in the method `clear()` instead.)

Reply:
Never mind this one: on second thought, I can see the case for not deleting the meta data unless clear() was called explicitly. (It is more consistent with RegisterView)